w3/protocol/api/controller/medical_record_patient_controller.go

```diff
@@ -0,0 +1,52 @@
+package controller
+
+import (
+	"halosuster/domain/usecase"
+	"halosuster/internal/helper"
+	"halosuster/protocol/api/dto"
+	"net/http"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+type iMedicalRecordPatientUsecase interface {
+	Create(req usecase.MedicalRecordPatientCreateRequest) (usecase.MedicalRecordPatientCreateResponse, error)
+}
+
+type IMedicalRecordPatientController interface {
+	Create(c *fiber.Ctx) error
+}
+
+type medicalRecordPatientController struct {
+	patientUsecase iMedicalRecordPatientUsecase
+}
+
+func NewMedicalRecordPatientController(patientUsecase iMedicalRecordPatientUsecase) *medicalRecordPatientController {
+	return &medicalRecordPatientController{patientUsecase}
+}
+
+func (ctr *medicalRecordPatientController) Create(c *fiber.Ctx) error {
+	var request usecase.MedicalRecordPatientCreateRequest
+
+	if err := c.BodyParser(&request); err != nil {
+		return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{
+			"error": "Cannot parse JSON",
+		})
```

Comment:
1. We already global error handler, just return fiber.StatusBadRequest (see: user_it_controller.go for usage example)

Reply:
done in next commit, forgot to merge the latest one.